src/composables/useUserPoolPercentage.ts

```diff
@@ -1,19 +1,33 @@
 import { bnum } from '@/lib/utils';
 import { Pool } from '@/services/pool/types';
-import { computed, Ref } from 'vue';
+import { Ref } from 'vue';
 import useNumbers from '@/composables/useNumbers';
 import { useTokens } from '@/providers/tokens.provider';
 import { usePoolStaking } from '@/providers/local/pool-staking.provider';
+import { isVeBalPool } from './usePool';
+import { useLock } from './useLock';
 
 export function useUserPoolPercentage(pool: Ref<Pool>) {
   const { balanceFor } = useTokens();
   const { stakedShares } = usePoolStaking();
+  // Avoid lock queries when pool is not veBAL:
+  const { lock, lockedFiatTotal } = useLock({
+    enabled: isVeBalPool(pool.value.id),
+  });
   const { fNum2 } = useNumbers();
 
+  const lockedAmount = computed(() => {
+    // return lock.value?.lockedAmount || '0';
```

Comment:
@garethfuller I started using lockedAmount but I think that I should use `lockedFiatTotal` instead, which is not a fiat, so probably better renaming it to `lockedBalanceTotal` or `lockedAmountTotal`. Am I wrong?

Reply:
Update: I saw that you already renamed it in another PR.